web/templates/user_display_only_pill.hbs

```diff
@@ -2,6 +2,7 @@
     <a data-user-id="{{user_id}}" class="view_user_profile pill" tabindex="0">
         {{#if img_src}}
         <img class="pill-image" src="{{img_src}}" />
+        <span class="pill-image-border"></span>
```

Comment:
I'd clean up the commit message here that you're introducing this empty element to align with others. It's not the class being added so much as the element, in other words.